### PR TITLE
Map `libsodium` extension to `sodium`

### PR DESCRIPTION
**Pull request by SerheyDolgushev**
1. Run `phpbrew ext install libsodium`
2. `~/.phpbrew/php/php-7.4.27/var/db/libsodium.ini` is created.

Expected content for `~/.phpbrew/php/php-7.4.27/var/db/libsodium.ini`:
```
extension=sodium.so
```

Actual content for `~/.phpbrew/php/php-7.4.27/var/db/libsodium.ini`:
```
extension=libsodium.so
```

### CODE REVIEW

**Comment by SerheyDolgushev** [comment]
@morozov can you please review this one?

**Comment by morozov** [comment]
@SerheyDolgushev sorry, I stopped using PHPBrew some time ago and cannot verify your patch.

**Comment by SerheyDolgushev** [comment]
@morozov can you please suggest correct reviewers for this one?

**Comment by morozov** [comment]
Maybe @jhdxr?

**Comment by SerheyDolgushev** [comment]
> Maybe @jhdxr?

Thank you! :)

**Comment by jhdxr** [comment]
this patch looks good to me. However, travis-ci is not working, while it's still required before merging. I don't have permission to bypass it, in other words, I cannot merge this or any other PR as well.

**Comment by morozov** [comment]
@jhdxr I made you an organization owner. You should be able to merge the PR now.

**Review by theofidry — APPROVED** [review]

**Comment by theofidry** [comment]
Closes #1193

**Review by peter279k — APPROVED** [review]

**Comment by ricklambrechts** [comment]
For me this works, thanks!

**Comment by theofidry** [comment]
I can confirm this fixes the issue.

**Comment by peter279k** [comment]
@c9s, could you look at this at your available time? Thanks!

**Review by c9s — APPROVED** [review]

**Comment by c9s** [comment]
@peter279k approved, let’s merge this

**Comment by peter279k** [comment]
@theofidry. Merged. Thanks.

**Pull request merged**
